### PR TITLE
Fix fortune on crops

### PR DESCRIPTION
**Pull request by diesieben07**
Vanilla increases the seed drops for crops when mined with fortune. This is broken in Forge.
[Reported on the forums](http://www.minecraftforge.net/forum/topic/58523-fortune-bug-with-crop-harvesting-1112/).

### CODE REVIEW

File: patches/minecraft/net/minecraft/block/BlockCrops.java.patch [hunk 1]
```diff
@@ -64,7 +64,8 @@
 +
      public void func_180653_a(World p_180653_1_, BlockPos p_180653_2_, IBlockState p_180653_3_, float p_180653_4_, int p_180653_5_)
      {
-         super.func_180653_a(p_180653_1_, p_180653_2_, p_180653_3_, p_180653_4_, 0);
```
Comment (LexManos) [inline patches/minecraft/net/minecraft/block/BlockCrops.java.patch:67]: Right, the 0 needs to be in the call to super.getDrops

Comment (diesieben07) [reply patches/minecraft/net/minecraft/block/BlockCrops.java.patch:67]: Doesn't change anything here, but for completeness sake, here you go.

**Comment by mezz** [comment]
Please create this PR for 1.12

**Comment by diesieben07** [comment]
Will this go into 1.11.x, too, or should I close this?

**Pull request merged**